Packs/Core/Playbooks/Playbook-Large_Upload.yml

```diff
@@ -0,0 +1,3868 @@
+      ResolveIP:
+        simple: "True"
+      SHA1:
+        complex:
+          root: File
+          accessor: SHA1
+          transformers:
+          - operator: uniq
+      SHA256:
+        complex:
+          root: PaloAltoNetworksXDR.OriginalAlert._all_events
+          accessor: os_actor_process_image_sha256
+          transformers:
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: PaloAltoNetworksXDR.OriginalAlert._all_events.actor_process_image_sha256
+                iscontext: true
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: PaloAltoNetworksXDR.OriginalAlert._all_events.causality_actor_process_image_sha256
+                iscontext: true
+          - operator: uniq
+      URL:
+        complex:
+          root: URL
+          accessor: Data
+          transformers:
+          - operator: uniq
+      URLSSLVerification:
+        simple: "False"
+      Username:
+        complex:
+          root: PaloAltoNetworksXDR.OriginalAlert._all_events.causality_actor_primary_normalized_user
+          accessor: username
+          transformers:
+          - operator: uniq
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1270
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: f7bd3ea7-e244-455e-8f36-7c451b94632d
+    type: title
+    task:
+      id: f7bd3ea7-e244-455e-8f36-7c451b94632d
+      version: -1
+      name: Analysis
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "11"
+      - "12"
+      - "13"
+      - "14"
+      - "17"
+      - "19"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1430
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: cd2a9c38-f9cc-424c-8bce-bbba04d2cd67
+    type: regular
+    task:
+      id: cd2a9c38-f9cc-424c-8bce-bbba04d2cd67
+      version: -1
+      name: Set Additional Alerts to Context
+      description: Set a value in context under the key you entered.
+      scriptName: Set
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "20"
+    scriptarguments:
+      key:
+        simple: Analysis.AdditionalAlerts
+      value:
+        complex:
+          root: PaloAltoNetworksXDR.OriginalAlert.alert_name
```

Comment:
same here - please change the root -> `Core.OriginalAlert.event`

Reply:
changed within all playbook